test/core/tests/card_timeout.py

```diff
@@ -10,6 +10,7 @@ class CardTimeoutTest(MetaflowTest):
 
     PRIORITY = 2
 
+    # fmt: off
```

Comment:
Can you add the equivalent `# fmt: on` after L:14?

Reply:
ah, didn't realize that was necessary, sry

looks like `# fmt: skip` at end of line now works (https://github.com/psf/black/pull/1800) so I did that, lmk what you think

Reply:
thanks!